README.md

```diff
@@ -47,158 +62,69 @@ just docker-build astria-sequencer debug
-just wait-for-celestia-jsonrpc-test-deployment
-```
-
-Then, get the Celestia JSON-RPC API token as follows:
-```sh
-# list pods
-kubectl get -n astria-celestia-jsonrpc-client-test pods
-# replace pod name with name printed above
-kubectl exec -n astria-celestia-jsonrpc-client-test pods/<your-pod-name-with-hashes-etc> -c celestia-bridge -- cat /home/celestia/.admin_token
-```
-
-Take note of this token, as the relayer step requires it.
-
-#### Start the relayer
+## Testing
 
-Pass your token from above to the `--celestia-bearer-token` flag.
+To run unit tests:
 
 ```sh
-./target/release/astria-sequencer-relayer --celestia-endpoint=http://astria-celestia-jsonrpc-client-test.localdev.me:80/jsonrpc/ --celestia-bearer-token=<token-from-above-step> --validator-key-file=$HOME/.cometbft/config/priv_validator_key.json 
+cargo test
 ```
 
-If Celestia is not running, pass the `--disable-writing` flag (and omit the celestia flags):
+Note that the `astria-proto` generates its code by running tests (and verifying
+that nothing changed). In order for its tests to run you also need
+[Buf](https://buf.build/docs/installation/) installed.
 
-```sh
-./target/release/astria-sequencer-relayer --validator-key-file=$HOME/.cometbft/config/priv_validator_key.json --disable-writing
-```
-
-#### Build and start astria go-ethereum
+## Formatting
 
-```sh
-git clone https://github.com/astriaorg/go-ethereum.git
-cd go-ethereum
-make geth
-./build/bin/geth --datadir ~/.astriageth/ init genesis.json
-./build/bin/geth --datadir ~/.astriageth/ --http --http.port=8545 --ws --ws.port=8545 --networkid=1337 --http.corsdomain='*' --ws.origins='*' --grpc --grpc.addr=localhost --grpc.port 50051
-```
+This project uses [rustfmt](https://github.com/rust-lang/rustfmt) to format rust
+sources, [taplo](https://github.com/tamasfe/taplo) to format toml files, and
+[markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) for
+markdown.
 
-#### Start the conductor
+### Rust
 
 ```sh
-./target/release/astria-conductor
+# Install rustfmt
+rustup +nightly-2023-07-07 component add rustfmt
+# Run rustfmt
+cargo +nightly-2023-07-07 fmt --all
 ```
 
-If Celestia is not running, pass the `--disable-finalization` flag:
+### Toml
 
 ```sh
-./target/release/astria-conductor --disable-finalization
-```
-
-#### Sending a transaction to the EVM rollup
-
-Import the following private key into Metamask or other Ethereum wallet:
-```
-0a6996ccaca77a1d48633da20062ae051e11c75f3f561dfd2ac01b0c9c874662
-```
-
-Alternatively, in the go-ethereum step, you can add genesis account allocations inside `go-ethereum/genesis.json` (you will need to re-init geth if you've already initialized it).
-
-Navigate to the `localhost:8545` network. Then, you can send transactions as normal. If you check the logs, you can see them move through the stack from sequencer -> relayer -> DA -> conductor -> geth.
-
-#### Clean up local environment
+# Install for macOS
+$ brew install taplo
```

Comment:
I think we should remove these `$`. It seems the rule was not correctly triggered because of the comments between the commands.

https://github.com/DavidAnson/markdownlint/blob/main/doc/md014.md